python/sdist/custom_commands.py

```diff
@@ -86,8 +86,24 @@ def build_libraries(self, libraries):
         import distutils.ccompiler
         distutils.ccompiler.CCompiler.compile = compile_parallel
 
+        # start new code
+        compilerType = self.compiler.compiler_type
+        for lib in libraries:
+            try: lib[1]['cflags'] = lib[1]['cflags'] + lib[1]['cflags_'+compilerType]
```

Comment:
add line break after colon here and below

Reply:
OK